__tests__/app.test.js

```diff
@@ -212,4 +262,12 @@ describe("Error Handling", ()=>{
                 expect(body.msg).toEqual('Bad Request')
             })
     })
+    it("should return status 400: Bad Request when given a review category that does not exist", () => {
+        return request(app)
+            .get('/api/reviews?category=invalid_category')
+            .expect(400)
+            .then(({body}) => {
+                expect(body.msg).toEqual('Bad Request')
+            })
+    })
```

Comment:
we want to add another test for this situation:
-what if there is a category that doesn't have any reviews yet?  In that case we want to send a 200 with an empty array